src/Adapter/Validate.php

```diff
@@ -135,4 +135,28 @@ public function isLinkRewrite($value)
     {
         return ValidateLegacy::isLinkRewrite($value);
     }
+
+    /**
+     * Check for a given email validity.
+     *
+     * @param string $email
+     *
+     * @return bool
+     */
+    public function isEmail(string $email): bool
+    {
+        return ValidateLegacy::isEmail($email);
+    }
+
+    /**
+     * Check if $data is a string.
+     *
+     * @param string $data Data to validate
+     *
+     * @return bool Validity is ok or not
+     */
+    public function isString(string $data): bool
```

Comment:
```suggestion
     * @param mixed $data Data to validate
     *
     * @return bool Validity is ok or not
     */
    public function isString($data): bool
```

As we want to test if it's a string, I'm not sure it's a good idea to add the type as parameter.
Otherwise the method seems useless :thinking:

Reply:
Used in `MultipleEmailsWithSeparatorValidator.php`